lib/matplotlib/axes/_axes.py

```diff
@@ -3796,7 +3796,7 @@ def dopatch(xs, ys, **kwargs):
     def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
                 vmin=None, vmax=None, alpha=None, linewidths=None,
                 verts=None, edgecolors=None,
-                **kwargs):
+                **kxwargs):
```

Comment:
It's `kwargs`.

If possible, please squash that last commit (and any others modifying this line) into the second commit.